<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="DESKTOP-2SJV0LG" failures="2" tests="5" name="MsSql" time="114.135" errors="0" timestamp="2022-10-28T10:39:32 IST">
  <testcase classname="testcases.ConnectorsTest" name="connectionTest" time="23.333">
    <failure type="org.openqa.selenium.TimeoutException" message="Expected condition failed: waiting for element found by By.xpath: //div[@class=&amp;apos;p-4 pb-0 testing-messages&amp;apos;]/div[3]/div/div/div to have text &amp;quot;Connection successful! You can go ahead and add your connector and create queries.&amp;quot;. Current text: &amp;quot;&amp;quot; (tried for 20 second(s) with 500 milliseconds interval)
Build info: version: &amp;apos;4.1.2&amp;apos;, revision: &amp;apos;9a5a329c5a&amp;apos;
System info: host: &amp;apos;DESKTOP-2SJV0LG&amp;apos;, ip: &amp;apos;192.168.1.14&amp;apos;, os.name: &amp;apos;Windows 10&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;17.0.4.1&amp;apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: true, browserName: chrome, browserVersion: 107.0.5304.88, chrome: {chromedriverVersion: 106.0.5249.61 (511755355844..., userDataDir: C:\Users\popst\AppData\Loca...}, goog:chromeOptions: {debuggerAddress: localhost:53123}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:53123/devtoo..., se:cdpVersion: 107.0.5304.88, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: cfd0b2f4dd44c15aaed720f7eaf3a3ad">
      <![CDATA[org.openqa.selenium.TimeoutException: Expected condition failed: waiting for element found by By.xpath: //div[@class='p-4 pb-0 testing-messages']/div[3]/div/div/div to have text "Connection successful! You can go ahead and add your connector and create queries.". Current text: "" (tried for 20 second(s) with 500 milliseconds interval)
Build info: version: '4.1.2', revision: '9a5a329c5a'
System info: host: 'DESKTOP-2SJV0LG', ip: '192.168.1.14', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.4.1'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: true, browserName: chrome, browserVersion: 107.0.5304.88, chrome: {chromedriverVersion: 106.0.5249.61 (511755355844..., userDataDir: C:\Users\popst\AppData\Loca...}, goog:chromeOptions: {debuggerAddress: localhost:53123}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:53123/devtoo..., se:cdpVersion: 107.0.5304.88, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: cfd0b2f4dd44c15aaed720f7eaf3a3ad
at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:138)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:231)
at testcases.ConnectorsTest.connectionTest(ConnectorsTest.java:124)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- connectionTest -->
  <testcase classname="testcases.ConnectorsTest" name="saveConnection" time="3.429">
    <skipped/>
  </testcase> <!-- saveConnection -->
  <testcase classname="testcases.ConnectorsTest" name="saveConnection" time="3.78">
    <skipped/>
  </testcase> <!-- saveConnection -->
  <testcase classname="testcases.ConnectorsTest" name="saveConnection" time="2.801">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;String.trim()&amp;quot; because &amp;quot;res&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "String.trim()" because "res" is null
at testcases.ConnectorsTest.saveConnection(ConnectorsTest.java:143)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- saveConnection -->
  <testcase classname="testcases.ConnectorsTest" name="addQuery" time="0.0">
    <skipped/>
  </testcase> <!-- addQuery -->
  <testcase classname="testcases.ConnectorsTest" name="connectionSaveApi" time="0.0">
    <ignored/>
  </testcase> <!-- connectionSaveApi -->
  <testcase classname="testcases.ConnectorsTest" name="connectionTestApi" time="0.0">
    <ignored/>
  </testcase> <!-- connectionTestApi -->
  <testcase classname="testcases.ConnectorsTest" name="connectionAddApi" time="0.0">
    <ignored/>
  </testcase> <!-- connectionAddApi -->
</testsuite> <!-- MsSql -->
